pass-manager-svc/pass-type/src/main/kotlin/com/example/passmanagersvc/passtype/application/port/out/PassTypeRepositoryOutPort.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.passmanagersvc.passtype.application.port.out
```

Comment:
<img width="685" alt="image" src="https://github.com/user-attachments/assets/56667bb3-def0-4c6b-9c99-1c20a62b7c86">


Choose one of approach:
1. `*.port.input` and `*.port.output`
2.`*.port.in` and `*.port.out`